### PR TITLE
added J to straight

### LINKED ISSUE #304: Add character: J to font: straight
*Character* : J
*Font* : straight

### LINKED ISSUE #305: Add character: K to font: straight
*Character* : K
*Font* : straight

### PR DESCRIPTION
**Pull request by komal7141**
*Describes your contribution*

closes #304 
closes #305

### CODE REVIEW

**Comment by komal7141** [comment]
#304 
#305

File: Fonts/Straight/straight.h [hunk 1]
```diff
@@ -114,27 +114,34 @@ class Straight : public Fonts
         return character;
     }
 
-    /*
+    
         char **J()
         {
             char **character = getCharGrid(ROWS,COLS);
```
Comment (AnshumanMahato) [inline Fonts/Straight/straight.h:120]: Pass the row and cols that you used to make your character

File: Fonts/Straight/straight.h [hunk 2]
```diff
@@ -114,27 +114,34 @@ class Straight : public Fonts
...
             return character;
         }
-    */
+    
 
-    /*
+    
         char **K()
         {
             char **character = getCharGrid(ROWS,COLS);
```
Comment (AnshumanMahato) [inline Fonts/Straight/straight.h:134]: Pass the row and cols that you used to make your character

**Review by AnshumanMahato — CHANGES_REQUESTED** [review]
Adjust the grid in a way so that it aligns with the rest of the characters

image

Having an empty row at the top should work I guess.

**Pull request closed without merging**